10_code/analysis.py

```diff
@@ -4,45 +4,49 @@
 
 fl = pd.read_csv("/Users/emma/Downloads/Finality Subsets/FL subsetfinalized.csv")
 wa = pd.read_csv("/Users/emma/Downloads/Finality Subsets/WA subsetfinalized.csv")
+FL_pre = pd.read_csv("/Users/emma/pds-2022-leep/20_intermediate_files/fl_pre.csv")
```

Comment:
these codes were added since we have new threshold subsets regarding overdoese missing values, maybe we want to make it more concise? concating florida all together and make all the three subsets have consistent column names